packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -317,6 +327,7 @@ export class Pipeline extends PipelineBase {
   private readonly _crossRegionSupport: { [region: string]: CrossRegionSupport } = {};
   private readonly _crossAccountSupport: { [account: string]: Stack } = {};
   private readonly crossAccountKeys: boolean;
+  private readonly enableKeyRotation: boolean;
```

Comment:
This should be `  private readonly enableKeyRotation?: boolean;`, to avoid changes to existing CloudFormation templates:

```suggestion
  private readonly enableKeyRotation?: boolean;
```